File: Server/Model/MenuItemModel.py

def MenuItemModel(item):
    return {       
        "Name": str(item["Name"]),
        "Cost": int(item["Cost"]),
        "Image": str(item["Image"]),
        "ItemID": int(item["ItemID"]),
        "Category": str(item["Category"]),
        "Description": str(item["Description"]),
    }

File: Client/src/app/MenuItemModel.ts
export class MenuItem {
    Cost : number;
    Image : string;
    ItemID : number;
    Name : string;
    Category : string;
    Description : string;

    constructor(obj:any){
        this.Cost = obj.Cost
        this.Image = obj.Image
        this.Name = obj.Name
        this.ItemID = obj.ItemID
        this.Category = obj.Category
        this.Description = obj.Description
    }
}
